packages/replay/package.json

```diff
@@ -7,14 +7,15 @@
   "types": "build/npm/types/index.d.ts",
   "sideEffects": false,
   "scripts": {
+    "build": "run-s build:worker && run-p build:core build:types build:bundle",
     "build:rollup": "run-s build:worker build:core",
-    "build": "run-s build:worker && run-p build:core build:types",
-    "build:dev": "run-s build #TODO adjust after adding CDN bundles",
+    "build:bundle": "rollup -c rollup.bundle.config.js",
+    "build:dev": "run-p build:worker build:rollup build:types",
     "build:worker": "rollup -c rollup.config.worker.js",
     "build:core": "rollup -c rollup.npm.config.js",
     "build:types": "tsc -p tsconfig.types.json",
-    "build:watch": "run-p \"build:worker --watch\" \"build:core --watch\" build:types:watch",
-    "build:dev:watch": "yarn build:watch #TODO adjust after adding CDN bundles",
+    "build:watch": "run-p \"build:worker --watch\" \"build:core --watch\" \"build:bundle --watch\" build:types:watch",
```

Comment:
Fair point. I initially wanted to avoid even more build scripts but I guess it's less confusing and more conforming to have individual ones. Also, once we tackled #6371 we'll be able to simplify these scripts a lot.  
[af20929](https://github.com/getsentry/sentry-javascript/pull/6391/commits/af209294d6b1bcb73a5eee3316b13a2e3f7b0478)